Experiments_toolkit.py

```diff
@@ -26,29 +20,28 @@ def create_train_test_validation(dictionary, pos_key, neg_key, percent_train, pe
             test -- dict (Dictionary with percent_test of original dictionary)
             validation -- dict (Dictionary with percent_validation of original dictionary)
         """
-    train = {pos_key: [], neg_key: []}
-    test = {pos_key: [], neg_key: []}
-    validation = {pos_key: [], neg_key: []}
+    train = {"postivies": [], "negatives": []}
+    test = {"postivies": [], "negatives": []}
+    validation = {"postivies": [], "negatives": []}
 
     if (percent_train + percent_test + percent_validation != 1):
         print("ERROR: percentages don't add to 1")
 
     train, test, validation = splitting_up_data(
-        dictionary, pos_key, train, percent_train, test, percent_test, validation, percent_validation)
+        positives, "positives", train, percent_train, test, percent_test, validation, percent_validation)
     train, test, validation = splitting_up_data(
-        dictionary, neg_key, train, percent_train, test, percent_test, validation, percent_validation)
+        negatives, "negatives", train, percent_train, test, percent_test, validation, percent_validation)
 
     return train, test, validation
 
 
-def splitting_up_data(dictionary, dict_key, train, percent_train, test, percent_test, validation, percent_validation):
+def splitting_up_data(data, key, train, percent_train, test, percent_test, validation, percent_validation):
```

Comment:
do you use percent_validation/

Reply:
No, but I included it just to make it mroe clear to the user how things will be split up and I check to make sure that all the percentages add up correctly